admin/zabbix/files/mac80211

```diff
@@ -1,7 +1,7 @@
-#see http://wiki.openwrt.org/doc/howto/zabbix for ready to use templates
+#see https://openwrt.org/docs/guide-user/services/network_monitoring/zabbix for ready to use templates
 
 # If you want to know the exact meaning of an UserParameter, you can search in the ieee80211 standard:
-# http://standards.ieee.org/getieee802/download/802.11-2012.pdf
+# https://standards.ieee.org/getieee802/download/802.11-2012.pdf
```

Comment:
404 page not found.

Reply:
Oops thanks catching this. I guess the new link should be https://standards.ieee.org/standard/802_11-2016.html (supersedes IEEE 802.11-2012) but I can't get past the paywall. Will need to update in master and 19.07 too.